sites/toonamiaftermath.com/toonamiaftermath.com.config.js

```diff
@@ -20,16 +20,16 @@ module.exports = {
 
     const playlist = playlists.find(p => date.isSame(p.startDate, 'day'))
 
-    return `https://api.toonamiaftermath.com/playlist?id=${playlist._id}&addInfo=true`
+    return `${API_ENDPOINT}/playlist?id=${playlist._id}&addInfo=true`
```

Comment:
Where's the mention of API_ENDPOINT ?

Reply:
It was already defined on line 6 and used on line 14, but line 23 had the URL hard-coded